lib/src/box.dart

```diff
@@ -17,14 +17,24 @@ class Box<T> {
   Store _store;
   Pointer<Void> _objectboxBox;
   var _entityDefinition, _entityReader, _entityBuilder, _fbManager;
+  var _idPropName;
 
   Box(this._store) {
     _entityDefinition = _store.getEntityModelDefinitionFromClass(T);
     _entityReader = _store.getEntityReaderFromClass<T>();
     _entityBuilder = _store.getEntityBuilderFromClass<T>();
     _fbManager = new OBXFlatbuffersManager<T>(_entityDefinition, _entityReader, _entityBuilder);
 
-    _objectboxBox = bindings.obx_box(_store.ptr, _entityDefinition["entity"]["id"]);
+    for (int i = 0; i < _entityDefinition["properties"].length; ++i) {
```

Comment:
not sth. box should be doing at runtime - id Property is defined by the model

Reply:
see my comment in model.dart

Reply:
Done.